MapboxGLAndroidSDK/src/main/java/com/mapbox/mapboxsdk/location/LocationComponent.java

```diff
@@ -739,14 +739,15 @@ public void zoomWhileTracking(double zoomLevel, long animationDuration,
                                 @Nullable MapboxMap.CancelableCallback callback) {
     checkActivationState();
     if (!isLayerReady) {
+      notifyUnsuccessfulCameraOperation(callback, null);
```

Comment:
Would it be useful to add a message for this occurrence as well? or would that spam the user?

Reply:
I'd rather skip it as this would happen only when component is paused, so it's expected that it'll ignored the updates.